docs/mesos/ManualInstallNetmodules.md

```diff
@@ -0,0 +1,59 @@
+# Manually Install Mesos + Netmodules
+This guide will walk you through building and installing Mesos + Net-Modules from source.
+
+## Background
+Netmodules and Mesos both make use of the `protobuf` library. To function correctly, Mesos and Netmodules must be built with identical compilations of `protobuf`. A standard Mesos installation will include bundled versions of this library, so we'll compile Mesos with an unbundled to ensure that netmodules is using precisely the same library as Mesos.
+
+## 1. Install Dependencies
+You can manually compile those libraries yourself, but we'll just install protobuf-devel from the `epel-release` repositories:
```

Comment:
@robbrockbank Sounds good. I'm thinking with those changes it might be good to nix the "Background" title and put move that content to be the first content in the "Install Dependencies" section. Agreed?